src/react/deckgl.js

```diff
@@ -109,11 +109,11 @@ export default class DeckGL extends React.Component {
   }
 
   _onRendererInitialized({gl, canvas}) {
-    gl.enable(GL.BLEND);
-    gl.blendFunc(GL.SRC_ALPHA, GL.ONE_MINUS_SRC_ALPHA);
-
-    // Enable polygon offset
-    gl.enable(GL.POLYGON_OFFSET_FILL);
+    setParameters(gl, {
+      blend: true,
+      blendFunc: [GL.SRC_ALPHA, GL.ONE_MINUS_SRC_ALPHA],
+      polygonOffsetFill: true
```

Comment:
Should we just move this into the draw setting you modified above, in layer.js, rather than flip it on globally?